sklearn/decomposition/fastica_.py

```diff
@@ -563,8 +565,7 @@ def inverse_transform(self, X, copy=True):
         """
         check_is_fitted(self, 'mixing_')
 
-        if copy:
```

Comment:
and why would we want to copy? ``fast_dot`` doesn't change inplace, right? That wouldn't make any sense.